docs/custom-music/index.md

```diff
@@ -132,3 +132,27 @@ Once done, save and repackage your mod, and you'll find your audio replacement p
 | AMB_MainMenu_Loop      | 259073131  | Played in the background of the Main Menu         |
 | PD3_MainTheme_Option02 | 1028929042 | Identical to 1033817546, but not used by the game |
 | PD3_MainTheme_Option02 | 1033817546 | Main menu theme used in game                      |
+
+### Correctly looping/making a menu music mod with a length superior to the original's
```

Comment:
This title is way too long, and shouldn't be nested under the heading for wwise ids.